Bento/Views/RequiringPreSizingLayoutPass.swift

```diff
@@ -0,0 +1,16 @@
+public protocol RequiringPreSizingLayoutPass {}
```

Comment:
@andersio can you please add documentation to this protocol? I think it's a protocol people may want to use.
I would describe there our use case

Reply:
Done